satpy/readers/satpy_cf_nc.py

```diff
@@ -294,8 +295,21 @@ def get_dataset(self, ds_id, ds_info):
         data.attrs.update(nc.attrs)  # For now add global attributes to all datasets
         if "orbital_parameters" in data.attrs:
             data.attrs["orbital_parameters"] = _str2dict(data.attrs["orbital_parameters"])
+
         return data
 
+    def get_area_def(self, dataset_id):
+        """Get area definition from CF complient netcdf."""
+        try:
+            area = AreaDefinition.from_cf(self.filename)
+            return area
+        except ValueError:
+            # No CF compliant projection information was found in the netcdf file or
+            # file contains 2D lat/lon arrays. To fall back to generating a SwathDefinition
+            # with the yaml_reader NotImplementedError is raised.
+            logger.warning("Failed to load AreaDefinition from nc file. Falling back to SwathDefinition.")
```

Comment:
I think this should be an info rather than a warning, and maybe reading "No AreaDefinition to load from nc file. Falling back to SwathDefinition."

Reply:
Ok I will change it.

Reply:
Just saw your comment above: yes it is falling back to SwathDefinition. I am embarrassed to say I can't remember if I added a unit test for this on another computer. I will check on that. But it is definitely tested manually ;-).

Reply:
I kind of feel this should be a DEBUG level. In the cases that the file/reader is customized with the coordinates that make up the longitude/latitude then there is no reason for any "extra" log messages. In these cases, this is expected behavior.

Reply:
Makes sense. @BENR0 sorry for asking for yet another change!

Reply:
No problem. It's an easy change ;-).